jobmatch/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Seed the Role database with possible user roles
+--
+MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS vals(a,b)
```

Comment:
Its the hsqldb equivalent to insert or update, and u can't drop because foreign key restrictions.

On January 25, 2016 5:58:33 PM CST, Charlie Hayes <notifications@github.com> wrote:
>> @@ -0,0 +1,15 @@
>> +--
>> +-- Seed the Role database with possible user roles
>> +--
>> +MERGE INTO ROLE AS t USING (VALUES(1,'user'), (2, 'company')) AS
>vals(a,b)
>
>I'm not familiar with MERGE; is this similar to INSERT_OR_UPDATE? Could
>we replace this with a simple DELETE FROM ROLE; INSERT ...; INSERT
>...;?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/idontusenumbers/JobMatch/pull/1/files#r50776875

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.